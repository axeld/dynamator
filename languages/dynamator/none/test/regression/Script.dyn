<dynamator language="none" suffix="html">
  <id name="x">
    <content>1</content>
  </id>
  <id name="y">
    <discard-tag/>
    <content>a</content>
  </id>
  <id name="z">
    <discard-tag/>
    <content>b</content>
  </id>
  <id name="alert">
    <content>
      alert("this was inserted");
    </content>
  </id>
</dynamator>
