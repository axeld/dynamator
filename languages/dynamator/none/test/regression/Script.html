<html>
<head>
  <title>Script test</title>
  <script type="text/javascript" language="javascript">
    x = 1;
    y = 2;
    if (x<y)
    {
        x = x&y;
    }
    if (y<x)
    {
        y&=x;
    }
  </script>
</head>
<body>
<p>Hi.</p>
  <script type="text/javascript" language="javascript">
    x  = <span id='x' class="DiscardTag"/>;
    yz = "<span id='y'/><span id='z'/>";
  </script>
  <script type="text/javascript" language="javascript" class="Discard">
  </script>
  <script type="text/javascript" language="javascript" id="alert">
  </script>
</body>
</html>
