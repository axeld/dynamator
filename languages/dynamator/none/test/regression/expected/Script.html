<html>
  <head>
    <title>Script test</title>
    <script type="text/javascript" language="javascript">
    x = 1;
    y = 2;
    if (x<y)
    {
        x = x&y;
    }
    if (y<x)
    {
        y&=x;
    }
  
</script>
  </head>
  <body>
    <p>Hi.</p>
    <script type="text/javascript" language="javascript">
    x  = 1;
    yz = "ab";
  
</script>
    <script type="text/javascript" id="alert" language="javascript">
      alert("this was inserted");
    </script>
  </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
